packages/code-studio/src/settings/ShortcutsSectionContent.tsx

```diff
@@ -0,0 +1,103 @@
+import React, { useState } from 'react';
+import classNames from 'classnames';
+import { Shortcut, Tooltip, ShortcutRegistry } from '@deephaven/components';
+import type { KeyState } from '@deephaven/components';
+import './ShortcutsSectionContent.scss';
+
+export default function ShortcutSectionContent(): JSX.Element[] {
+  function formatCategoryName(name: string): string {
+    return name
```

Comment:
I thought about putting it in a utils, but felt it wasn't really robust enough. It only converts screaming snake case to title case. And it's kind of a fake title case since it doesn't check for things like "a", "the", "and" which would be lowercase in title case